firmware/Core/Src/telecommands/telecommand_executor.c

```diff
@@ -14,6 +14,9 @@
 #include <stdbool.h>
 
 
+uint32_t total_tcmd_queued_count = 0;
+uint64_t most_recent_tcmd_timestamp_sent = 0;
```

Comment:
What does "most recent" mean? Most recent executed? Most recent received? Something else?

Proposal: rename to `latest_received_tcmd_timestamp_sent`